### PR TITLE
Add the port forward option in the instruction on how to provision an…

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Add the port forward option to step 2 in the instruction on how to provision an SKR.
- Fix a typo.

**Related issue(s)**
See #676

### CODE REVIEW

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
+Alternatively, you can perform `kubectl port-forward` on the chosen pod to expose it on your local machine. Assuming that KEB is installed in the default namespace, expose it to port `8080`:
```
Comment (piotrmiskiewicz) [inline docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:30]: KEB is always installed in the "kcp-system" namespace, so the command is:
```
kubectl port-forward -n kcp-system deployments/kcp-kyma-environment-broker 8080
```

**Review by piotrmiskiewicz — APPROVED** [review]

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md [hunk 2]
```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
-
-3. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
+Alternatively, you can perform `kubectl port-forward` on the chosen pod to expose it on your local machine. Expose it to port `8080`:
```
Comment (majakurcius) [inline docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:30]:
```suggestion
Alternatively, you can perform `kubectl port-forward` on the chosen Pod to expose it on your local machine. Expose it on port `8080`:
```
- According to our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#camelcase), we capitalized `Pod`. 
- `To` or `on` port? Can you check with a dev?

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md [hunk 3]
```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
...
+```bash
+  kubectl port-forward -n kcp-system deployments/kcp-kyma-environment-broker 8080
+```
+You should get information about the successful forwarding. For example:
```
Comment (majakurcius) [inline docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:34]:
```suggestion
As a result, you get information about the successful forwarding. For example:  
```

Try not to use `should`, as it leaves too much room for ambiguity: `should`, but there's an option you get something else? What then? 

Also, I wouldn't say this is necessary to include this result here at all.

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md [hunk 4]
```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
...
+```bash
+  kubectl port-forward -n kcp-system deployments/kcp-kyma-environment-broker 8080
+```
+You should get information about the successful forwarding. For example:  
+```bash
+  Forwarding from 127.0.0.1:8080 -> 8080
+  Forwarding from [::1]:8080 -> 8080
+```  
+
+1. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
```
Comment (majakurcius) [inline docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:40]: Why do we have step 1 after step 2? 
If it's part of this "alternative", then we should think about how to restructure this section, for example, put the whole section in two [toggle panels ](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/05-documentation-toggle.md)for two alternatives.

File: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md (hunk 3 above)
Comment (IwonaLanger) [reply docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md:34]: Tbh I practically copied it from another doc, but - sure - there's always scope for improvement :)

**Review by majakurcius — APPROVED** [review]

**Pull request merged**